frontend/public/js/form.js

```diff
@@ -35,6 +35,13 @@ $(document).ready(
             $("#custom-font").css("display", "none");
           }
         });
+        $("#text-color").click(function(){
```

Comment:
you need to remove above jquery colde , which also mentions `text-color`, see line 28.

Reply:
@rupav Thanks. I made the changes :)